src/ResourceManager/EventHub/Commands.EventHub/help/Remove-AzureRmEventHub.md

```diff
@@ -29,13 +42,46 @@ PS C:\> Remove-AzureRmEventHub -ResourceGroupName MyResourceGroupName -Namespace
 
 Removes the Event Hub \`MyEventHubName\`.
 
+### Example 2 - InputObject
+```
+PS C:\> Remove-AzureRmEventHub -InputObject $inputobject
```

Comment:
@v-Ajnava [same comment as the ServiceBus PR](https://github.com/Azure/azure-powershell/pull/6732#discussion_r204582965) -- for examples using `-InputObject`, please make sure to show the user where to get the value for the parameter

Reply:
resolved, updated examples with variable and Piping scenario